### DISCUSSION TITLE
Unable to pull the trivy db image

### DISCUSSION THREAD

**Discussion by idrissneumann** [aquasecurity/trivy#7865] [Bugs]
### Description

The GCR registry seems down.

### Desired Behavior

Beiing able to pull the image

### Actual Behavior

"Too many request" even if we make a single docker pull

### Reproduction Steps

```bash
Just run this command:


ineumann ~ $ docker pull ghcr.io/aquasecurity/trivy-db:2
Error response from daemon: toomanyrequests: retry-after: 41.127µs, allowed: 44000/minute
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

None

### Mode

None

### Debug Output

```bash
Just run this command:


ineumann ~ $ docker pull ghcr.io/aquasecurity/trivy-db:2
Error response from daemon: toomanyrequests: retry-after: 41.127µs, allowed: 44000/minute
```

### Operating System

Everyone (macos, linux, etc)

### Version

```bash
trivy-db:2
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by nikpivkin** [comment]
Duplicate of https://github.com/aquasecurity/trivy/discussions/7538

**Reply by simar7** [reply]
@idrissneumann please see https://github.com/aquasecurity/trivy/discussions/7699
